web/src/components/alerts/index.tsx

```diff
@@ -3,5 +3,5 @@
  * SPDX-License-Identifier: GPL-2.0-or-later
  */
 
-export * from "./ErrorPage";
-export * from "./Warning";
+export { ErrorPage } from './ErrorPage';
+export { WarningAlert } from "./Warning";
```

Comment:
Any benefit of doing it this way?

Reply:
Same as above reasoning.

re: https://github.com/ArnaudBarre/eslint-plugin-react-refresh?tab=readme-ov-file#limitations